Lullabot/lb-ebook-poc
# 58 Menu Scroll

Resolves #58 

To test:

- visit https://pr60-nxe2ohmviolvujkjon0bzf9d32qn7pr3.tugboatqa.com/ebooks/large-team-challenges/chapters/chapter-4
- open menu, ensure menu scrolls independently of the rest of the page
